### ISSUE TITLE
Backend: Message endpoint should support hasBeenSeen

### ISSUE THREAD

**Issue by benna100** [HackYourFuture-CPH/chattie#138]
## User story

**Who:** **As a** developer

**What:** **I want to** be able to get, update and create message with hasBeenSeen

**Why:** **so that we can** work with unread messages

## Implementation details
- This story is dependent on [this story](https://github.com/HackYourFuture-CPH/chattie/issues/137)
- The hasBeenSeen should when a message is created be set to false
- When a message is gotten either from `/messages` or `/messages/3` remember to include the hasBeenSeen here!
- When a message is updated using patch this should support hasBeenSeen

Basically whenever we work with messages it should be possible to use hasBeenSeen. There might be cases i have forgotten here, so really think hard about this problem!

**Comment by benna100** [comment]
Did not think
